crates/bevy_ecs/src/relationship/related_methods.rs

```diff
@@ -168,7 +178,10 @@ impl<'w> EntityWorldMut<'w> {
                 // SAFETY: We'll manually be adjusting the contents of the parent to fit the final state.
```

Comment:
There's a final mention of "parent" that I can see in the surrounding lines.
Isn't part of the changes from this PR but could be fixed, still.